processor/test/services/adyen-payment.service.spec.ts

```diff
@@ -42,17 +46,11 @@ import { PaymentResponse } from '@adyen/api-library/lib/src/typings/checkout/pay
 import { KlarnaDetails } from '@adyen/api-library/lib/src/typings/checkout/klarnaDetails';
 import { CardDetails } from '@adyen/api-library/lib/src/typings/checkout/cardDetails';
 import { ApplePayDetails } from '@adyen/api-library/lib/src/typings/checkout/applePayDetails';
-import { ApplePayPaymentSessionError } from '../../src/errors/adyen-api.error';
 
 interface FlexibleConfig {
   [key: string]: string; // Adjust the type according to your config values
 }
 
-export type AdyenPaymentServiceOptions = {
-  ctCartService: CommercetoolsCartService;
-  ctPaymentService: CommercetoolsPaymentService;
-};
```

Comment:
We redeclared the service options which imo was not needed caus we already have that in the `adyen-payment.service` so I removed it from here.